Sources/TuistEnvKit/GoogleCloudStorage/GoogleCloudStorageClient.swift

```diff
@@ -41,6 +49,30 @@ public final class GoogleCloudStorageClient: GoogleCloudStorageClienting {
         self.urlSessionScheduler = urlSessionScheduler
     }
 
+    func latestTuistEnvBundleURL() -> Foundation.URL {
+        GoogleCloudStorageClient.url(releasesPath: "latest/tuistenv.zip")
+    }
+
+    func tuistBundleURL(version: String) -> Observable<Foundation.URL?> {
+        let releaseURL = GoogleCloudStorageClient.url(releasesPath: "\(version)/tuist.zip")
+        var releaseRequest = URLRequest(url: releaseURL)
+        releaseRequest.httpMethod = "HEAD"
+        let releaseSingle = urlSessionScheduler.single(request: releaseRequest)
+
+        let buildURL = GoogleCloudStorageClient.url(buildsPath: "tuist-\(version).zip")
```

Comment:
Note that this makes it easier to try builds from commits because we no longer need to compile the source code of the commit. We just pull the binary from GCS and that's it. In the future I'd like to support pinning the environment to a branch, and then make sure that we are always pointing to the latest commit in that branch.